abmodel/tests/models/test_population.py

```diff
@@ -1,14 +1,30 @@
-from unittest import TestCase
+import pytest
+
 
 from abmodel.models.population import BoxSize
 
 
-class BoxSizeTestCase(TestCase):
+class TestBoxSizeCase():
```

Comment:
Remove  unnecessary `()`. 
In a class declaration, you should avoid empty parentheses `()`.